getting_started/dedicated_administrators.adoc

```diff
@@ -0,0 +1,166 @@
+*RoleBinding[dedicated-project-admin]* for the project:
+
+====
+----
+$ oc describe policyBindings :default -n <project_name>
+
+Name:					:default
+Created:				2 weeks ago
+Labels:					<none>
+Annotations:				<none>
+Last Modified:				2016-05-03 05:34:49 -0400 EDT
+Policy:					<none>
+RoleBinding[admin]:
+					Role:			admin
+					Users:			fred@example.com <1>
+					Groups:			<none>
+					ServiceAccounts:	<none>
+					Subjects:		<none>
+RoleBinding[dedicated-project-admin]:
+					Role:			dedicated-project-admin
+					Users:			alice@example.com, bob@example.com <2>
+					Groups:			<none>
+					ServiceAccounts:	<none>
+					Subjects:		<none>
+...
+----
+<1> The *fred@example.com* user is a normal, project-scoped administrator of this
+project.
+<2> The *alice@example.com* and *bob@example.com* users are cluster administrators.
+====
+
+To view details on your increased permissions, and the sets of
+link:../architecture/additional_concepts/authorization.html#evaluating-authorization[verbs
+and resources] associated with the *dedicated-cluster-admin* and
+*dedicated-project-admin* roles, run the following:
+
+----
+$ oc describe clusterrole/dedicated-cluster-admin
+$ oc describe clusterrole/dedicated-project-admin
+----
+
+[[gs-dedicated-admin-granting-permissions]]
+== Granting Permissions to Users or Groups
+
+To grant permissions to other
+link:../architecture/additional_concepts/authentication.html#users-and-groups[users
+or groups], you can add, or _bind_, a role to them using the following commands:
+
+----
+$ oadm policy add-role-to-user <role> <user_name>
+$ oadm policy add-role-to-group <role> <group_name>
+----
+
+See link:../admin_guide/manage_authorization_policy.html[Managing Authorization
+Policies] for more details on these and related authorization tasks.
+
+[[gs-dedicated-admin-creating-service-accounts]]
+== Creating Service Accounts
+
+You can create a
+link:../architecture/core_concepts/projects_and_users.html#users[service
+account] to be able to run applications like Jenkins that make calls back to
+{product-title}.
+
+See the link:../dev_guide/service_accounts.html[Developer Guide] for basic
+service account management tasks, which as a cluster administrator you can
+perform in any user-created project, and see
+link:../admin_guide/service_accounts.html[Configuring Service Accounts] for more
+advanced, cluster-wide settings.
+
+[[gs-dedicated-admin-adding-or-removing-default-image-streams-and-templates]]
+== Adding or Removing Default Image Streams and Templates
+
+See link:../admin_guide/osd_imagestreams_templates.html[Managing the Default
+Image Streams and Templates] for information on the core set of image streams
+and templates provided by default in the global *openshift* project. You can
+also modify, add, or remove these objects from the *openshift* project as a
+cluster administrator.
+
+[[gs-dedicated-admin-setting-quotas-and-limit-ranges]]
+== Adjusting Quotas and Limit Ranges and Setting Defaults
```

Comment:
Yeah, I ended up merging two sections into this, and the title suffered. Can re-title.